drivers/cyberpower-mib.c

```diff
@@ -24,20 +24,26 @@
 
 #include "cyberpower-mib.h"
 
-#define CYBERPOWER_MIB_VERSION		"0.5"
+#define CYBERPOWER_MIB_VERSION		"0.51"
 #define CYBERPOWER_OID_MODEL_NAME	".1.3.6.1.4.1.3808.1.1.1.1.1.1.0"
 
 /* CPS-MIB::ups */
 #define CYBERPOWER_SYSOID			".1.3.6.1.4.1.3808.1.1.1"
 
+/* https://www.cyberpowersystems.com/products/software/mib-files/ */
+/* Per CPS MIB 2.9 upsBaseOutputStatus OBJECT-TYPE: */
 static info_lkp_t cyberpower_power_status[] = {
-	{ 2, "OL", NULL, NULL },
-	{ 3, "OB", NULL, NULL },
-	{ 4, "OL BOOST", NULL, NULL },
-	{ 5, "OFF", NULL, NULL },
-	{ 7, "OL", NULL, NULL },
-	{ 1, "NULL", NULL, NULL },
-	{ 6, "OFF", NULL, NULL },
+	{ 1, "NULL", NULL, NULL },	/* unknown */
```

Comment:
@aquette : seems "NULL" is not a known status string; do we have something useful for "unknown"? Empty string?..